README.md

```diff
@@ -1,16 +1,34 @@
 # Multi-threaded Google  Ban Checker
 
+
+[![Contributors](https://img.shields.io/github/contributors/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/blob/main/Contributors.md)
+[![Issues](https://img.shields.io/github/issues/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/issues)
+[![Open Pull Requests](https://img.shields.io/github/issues-pr-raw/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/pulls)
+[![Last Commit](https://img.shields.io/github/last-commit/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/commits/main)
+[![Stars](https://img.shields.io/github/stars/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/stargazers)
+[![Forks](https://img.shields.io/github/forks/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/network/members)
+[![Watchers](https://img.shields.io/github/watchers/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/watchers)
+[![License](https://img.shields.io/github/license/tuladhar/multi-threaded-google-ban-checker)](https://github.com/tuladhar/multi-threaded-google-ban-checker/blob/main/LICENSE)
+
```

Comment:
these are not the type of links you think. they are API requests to  your repository stats
![Screen Shot 2020-10-19 at 12 45 44 PM](https://user-images.githubusercontent.com/5104144/96504143-26ce6c00-1209-11eb-837f-2481e9a5bbcb.png)